### PR TITLE
Add VPN moments page (Fixes #13303)

### LINKED ISSUE #13303: Moments page for VPN
## Description

Figma: https://www.figma.com/file/UFZ8R9V3lVxpZFQ34g0fN9/VPN-Moments-%26-Email?type=design&node-id=1412-4
Copy doc: https://docs.google.com/document/d/1Md6A8wCr1O_fbOUK2qtNl8JAPSr0_vFxvXIbEIQtSXQ/edit?usp=sharing

Target date: July 6th
A/B test: 50/50 for each locale
Languages: English, German, French

---

#### Success Criteria
- [ ] Page is live for July 6th

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

- Adds `/firefox/welcome/15/` page for `/en/`, `/de/`, and `/fr/` locales.
- Adds traffic cop A/B test for 2 variations per locale.

## Issue / Bugzilla link

#13303

## Testing

- http://localhost:8000/en-US/firefox/welcome/15/?v=1
- http://localhost:8000/en-US/firefox/welcome/15/?v=2
- http://localhost:8000/en-CA/firefox/welcome/15/?v=1
- http://localhost:8000/en-CA/firefox/welcome/15/?v=2
- http://localhost:8000/en-GB/firefox/welcome/15/?v=1
- http://localhost:8000/en-GB/firefox/welcome/15/?v=2
- http://localhost:8000/de/firefox/welcome/15/?v=1
- http://localhost:8000/de/firefox/welcome/15/?v=2
- http://localhost:8000/fr/firefox/welcome/15/?v=1
- http://localhost:8000/fr/firefox/welcome/15/?v=2

### CODE REVIEW

File: bedrock/firefox/templates/firefox/welcome/page15/base.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page15') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-15' %}
+{% set _utm_campaign = 'welcome-15-vpn' %}
+
+{% block experiments %}
+  {% if switch('welcome-15-experiment', ['en-US', 'de', 'fr']) %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/welcome/page15/base.html:24]: Need to include en-CA and en-GB here. I think doing just `'en'` covers all three.

File: bedrock/firefox/templates/firefox/welcome/page15/base.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  ) }}
+{% endblock %}
+
+{% block shoulder_cta %}{% endblock %}
+
+{% block content_intro %}
+    {% if variation == '2' %}
+      <div class="c-main variation-2 mzp-l-content mzp-t-content-lg">
+        <img class="c-main-logo" src="/media/protocol/img/logos/mozilla/vpn/logo-flat.svg" width="60" alt="">
+        <h1 class="c-main-title">{{ var_1_main_title }}</h1>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/welcome/page15/base.html:61]: Sorry, just noticed all the strings are labeled "var_1" but this is actually `variation == 2` above. Should they be switched around? Just to make sure we have the right text on the right variant.

File: bedrock/firefox/templates/firefox/welcome/page15/base.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+          optional_attributes= {
+                'data-cta-text' : 'Try Mozilla VPN',
+                'data-cta-type' : 'button'
+          }
+        )}}
+      </div>
+    {% else %}
+      <div class="c-main variation-1 mzp-l-content mzp-t-content-md">
+        <img class="c-main-logo" src="/media/protocol/img/logos/mozilla/vpn/logo-flat.svg" width="60" alt="">
+        <h1 class="c-main-title">{{ var_2_main_title }}</h1>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/welcome/page15/base.html:79]: A minor thing but I can anticipate a designer complaining about the text wrap on the v1 headline in en-US and en-CA... maybe this needs a max-width added? v2 has a narrower container already so it's fine.

image

**Review by craigcook — CHANGES_REQUESTED** [review]
Sorry, noticed a few other things on a second pass.

File: bedrock/firefox/templates/firefox/welcome/base.html [hunk 4]
```diff
@@ -21,7 +21,9 @@
 <main class="mzp-t-firefox">
   <header class="c-page-header">
     <div class="mzp-l-content c-page-header-inner">
-      <img src="{{ static('protocol/img/logos/firefox/browser/logo-word-hor.svg') }}" alt="Firefox" width="216" height="40" class="c-header-logo">
+      {% block shoulder_logo %}
+        <img src="{{ static('protocol/img/logos/firefox/browser/logo-word-hor.svg') }}" alt="Firefox" width="216" height="40" class="c-header-logo">
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/base.html:25]: Gonna punt on this one

File: bedrock/firefox/templates/firefox/welcome/page15/base.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page15/base.html:24]: Good catch!

File: bedrock/firefox/templates/firefox/welcome/page15/base.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page15/base.html:61]: Doh, will fix!

**Review by craigcook — APPROVED** [review]

**Pull request merged**